akka-actor/src/main/boilerplate/akka/japi/tuple/Tuples.scala.template

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.japi.tuple
+
+[3..22#/**
+ * Used to create tuples with 1 elements in Java.
+ */
+object Tuple1 {
+  def create[[#T1#]]([#t1: T1#]) = new Tuple1([#t1#])
+}
+
+/**
+ * Java API Tuple container.
+ */
+@SerialVersionUID(##1L)
+final case class Tuple1[[#T1#]]([#t1: T1#]) extends scala.Product with scala.Serializable {
```

Comment:
you're right, its not required because it's a case class - I copied it over from another PR. Removed now.